library/core/src/char/methods.rs

```diff
@@ -927,7 +927,7 @@ impl char {
     #[must_use]
     #[inline]
     pub(crate) fn is_grapheme_extended(self) -> bool {
-        unicode::Grapheme_Extend(self)
+        self > '\x7f' && unicode::Grapheme_Extend(self)
```

Comment:
Any reason not to instead update `unicode-table-generator` so this becomes a part of `Grapheme_Extend` (`unicode::unicode_data::grapheme_extended::lookup`)? So as to help anywhere else this call might wind up getting used

Reply:
We really should have a `const ASCII_MAX: char = '\x7f'` somewhere, maybe even public. We throw that magic number around a lot

Reply:
The generator doesn't know specifics about particular properties yet, AFAICS, but I could imagine it could generically optimize lookups outside of a property's min/max.

Reply:
Here you go: https://github.com/rust-lang/rust/pull/122013